src/desktop/net/ProtocolProxy.ts

```diff
@@ -5,26 +5,46 @@ import { Session } from "electron"
 import { errorToObj } from "../../api/common/threading/MessageDispatcher.js"
 import { lazyMemoized } from "@tutao/tutanota-utils"
 import { getMimeTypeForFile } from "../files/DesktopFileFacade.js"
-import { ServiceUnavailableError } from "../../api/common/error/RestError.js"
+import { Agent, fetch, RequestInfo as UndiciRequestInfo, RequestInit as UndiciRequestInit } from "undici"
+
+type GlobalFetch = typeof global.fetch
 
 const TAG = "[ProtocolProxy]"
 
 export const ASSET_PROTOCOL = "asset"
-const PROXIED_REQUEST_READ_TIMEOUT = 20000
+
+/** How long the socket should stay open without any data sent over it. See IDLE_TIMEOUT_MS in tutadb. */
+const SOCKET_IDLE_TIMEOUT_MS = 5 * 60 * 10_000 + 1000
```

Comment:
It's bigger than the server's IDLE_TIMEOUT, is it correct?

Reply:
Server idle timeout might also change but yes generally we want to let the server drive this one